tests/ts_sql_eqc_util.erl

```diff
@@ -0,0 +1,165 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+%% @doc Convenience functions to generate DDLs for up-/downgrade
+%%      testing purposes.
+
+-module(ts_sql_eqc_util).
+
+-compile(export_all).
+
+-ifdef(EQC).
+-include_lib("eqc/include/eqc.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("../riak/deps/riak_ql/include/riak_ql_ddl.hrl").
```

Comment:
We do not want to create a hard-dependency upon `riak_ql` at compile time, due to how we don't lock `riak_test` modules.  We should add some testing functions to QL to allow it to be linked in at run time.  Happy to add these as necessary.